modin/core/dataframe/pandas/partitioning/axis_partition.py

```diff
@@ -157,7 +157,9 @@ def deploy_axis_func(
         lengths = kwargs.pop("_lengths", None)
 
         dataframe = pandas.concat(list(partitions), axis=axis, copy=False)
-        result = func(dataframe, **kwargs)
+        # To not mix the args for deploy_axis_func and args for func, we fold
+        # args into kwargs. This is a bit of a hack, but it works.
```

Comment:
This is a really hack that should probably be fixed in #4473.

Reply:
Agreed. This is a quick hack that will be addressed in that PR.